bitnami/oauth2-proxy/Chart.yaml

```
@@ -30,4 +30,4 @@ name: oauth2-proxy
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/oauth2-proxy
   - https://github.com/oauth2-proxy/oauth2-proxy
-version: 3.0.4
+version: 3.0.5
```

Since this is adding a new feature, please increase the minor as well, e.g.:

```suggestion
version: 3.1.0
```